### PR TITLE
Fix desktop nav scroll

### PR DESCRIPTION
**Pull request by alexBinns**
This PR fixes the nav bar on desktop resetting whenever a page is loaded.

Testing steps:

- [ ] Navigate to a page on docs site
- [ ] Scroll the navigation on desktop
- [ ] Click a link in the navigation
- [ ] See the the scroll level isn't reset

### CODE REVIEW

**Comment by MeganSteenkamp** [comment]
This looks good overall, nice 🥳 
Looks like it needs some linting and removing of console statements.
I made a PR that might help: https://github.com/centrapay/centrapay-docs/pull/920

File: src/components/AppShell.astro [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+import Header from './Header.vue';
+import DesktopNavigation from './DesktopNavigation.astro';
+const { path, navigation } = Astro.props;
+
+const navPath = path.endsWith('/') ? path.slice(0, -1) : path;
+---
+
+  <div class="flex h-screen w-screen">
```
Comment (MeganSteenkamp) [inline src/components/AppShell.astro:9]: @alexBinns are you missing a closing div? 🤨

File: src/components/DesktopNavigation.astro [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+              />
+          ))
+        }
+      </ul>
+    </nav>
+  </div>
+</div>
+
+<script is:inline>
+  {
```
Comment (MeganSteenkamp) [inline src/components/DesktopNavigation.astro:38]: Suggestion: Remove brackets

File: src/components/AppShell.astro (hunk 1 above)
Comment (alexBinns) [reply src/components/AppShell.astro]: You are right...fixed

File: src/components/DesktopNavigation.astro (hunk 2 above)
Comment (alexBinns) [reply src/components/DesktopNavigation.astro]: Done

File: src/components/AppShell.astro [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+import Header from './Header.vue';
+import DesktopNavigation from './DesktopNavigation.astro';
+const { path, navigation } = Astro.props;
+
+const navPath = path.endsWith('/') ? path.slice(0, -1) : path;
+---
+
+<div class="flex h-screen w-screen">
+  <Header client:load path={path} navigation={navigation} />
```
Comment (MeganSteenkamp) [inline src/components/AppShell.astro:10]: @alexBinns should the the path here be `navPath`?

**Review by MeganSteenkamp — APPROVED** [review]

File: src/components/AppShell.astro (hunk 3 above)
Comment (alexBinns) [reply src/components/AppShell.astro]: Yup

**Pull request merged**